README.md

```diff
@@ -1,13 +1,137 @@
-erl_optics
-=====
+# erl_optics
 
-An OTP library
+An OTP application for [optics](https://github.com/RAttab/optics).
 
-### Requirements
+## Requirements
+-----
 
 * Erlang OTP >= 21
+* [optics](https://github.com/RAttab/optics) v1.1.10
 
-Build
+## Building
 -----
-
     $ rebar3 compile
+
+## Usage
+-----
+
+For now, the only working mode is Carbon.
+
+need env :
+```
+  {erl_optics, [
+      {hostname, "https://my-carbon-host.com"},
+      {port, 2003},
+      {mode, carbon},
+      {interval, 10000}
+  ]}
+```
+
+To start the application:
+```
+  erl_optics_app:start(),
+```
+
+To start the optics polling:
+```
+  erl_optics:start_optics(<<"my_namespace">>),
+```
+or
+```
+  erl_optics:start_optics(<<"my_namespace">>, Lenses),
+```
+
+The namespace will be prepending the name of every metric, for
+example: `"optics_namespace.my_counter"`
```

Comment:
In the code above you use `"my_namespace"`, so I'd rename this example to `"my_namespace.my_counter"`.